README.md

```diff
@@ -72,14 +72,25 @@ with `npm test`
 Python tests of the release scripts are in `/scripts/release/tests` and can be run from the top level `orderly-web`
 directory by running `./scripts/release/tests/test-release.sh`
 
-### Run linter
-To run [detekt](https://detekt.github.io/detekt/index.html) against the main code (i.e. excluding tests):
+### Code linting
+All new code should follow our [code style conventions]. We have [settings files] that configure IDEA to format code
```

Comment:
May be worth mentioning the ways this differs from the default - primarily the way braces are treated - as this is what causes trouble!

Reply:
Good shout - done!